src/cli/bootstrap/index.ts

```diff
@@ -389,29 +390,34 @@ async function updateAccountControllers(govKeys: string[], newPublicKey: PublicK
     console.log('Change the key of the accounts to the new key', newPublicKey.toString());
     await updateAccountKey('found.tmy', newPublicKey);
 
+    function updateControlByOptions(account: string) {
+        return { addCodePermission: addCodePermissionTo.includes(account) ? 'vesting.tmy' : undefined };
+    }
```

Comment:
why u added this?

Reply:
this is needed to be able to execute the `eosio.token::transfer()` function as an in-line action here https://github.com/Tonomy-Foundation/Tonomy-Contracts/pull/114/files#diff-4e30cce0e778fda610641abde6c17dc6b99db3adafc796ea10024484629976bcR68